docs/topics/maven-run.adoc

```diff
@@ -34,11 +34,26 @@ To run the {MavenName} perform the following steps.
 +
 The above example demonstrates the minimum required arguments. See xref:maven_arguments[{ProductShortName} Maven Arguments] for a detailed description of all available arguments.
 
+. If using Java 11, then `--add-modules=java.se` must be added to the `MAVEN_OPTS` argument. When using other versions of Java this is not necessary, and you can proceed to the next step.
```

Comment:
well, MAVEN_OPTS is not an argument but an environment variable.
Also I would say "when using older versions of java" as we dont know if with Java 12 this will happen the same

Reply:
I've made both of these corrections, and the merge request has been updated.